blog/2024-07-19-coordinator_async_setup.md

```diff
@@ -0,0 +1,78 @@
+---
+author: Josef Zweck
+authorURL: https://github.com/zweckj
+title: Set up your `DataUpdateCoordinator` with `_async_setup`
+---
+
+In Home Assistant 2024.8, we are introducing the `_async_setup` method for the data update coordinator. 
+This method allows you to run asynchronous code to prepare your `DataUpdateCoordinator` instance
+or to load data that only needs to be loaded once.
+
+You can override `_async_setup` in your coordinator, and it will be automatically
```

Comment:
Should we add something that besides overwriting this method the dev can also instead define their own setup method (and also in the example below clarify that it's for overwriting the method).

Reply:
Mmh not sure about the setup_method. I expect it will hardly be used. Like, I never see the update method being used. Imo that's more interesting for a proper docs update of the coordinator page, not the blog. 
I don't know, what would you put there? I would've thought it's clear enough.

Reply:
now got what you meant. Since I found some `update_method` params after all, decided to add it and changed the comment.